build-support/python/make-mesos-native-egg

```diff
@@ -139,38 +139,33 @@ CENTOS6_X86_64_DEPENDENCIES=(
   apr-devel
   cyrus-sasl-devel
   cyrus-sasl-md5
-  devtoolset-2-binutils
-  devtoolset-2-gcc
-  devtoolset-2-gcc-c++
+  devtoolset-7-binutils
+  devtoolset-7-gcc
+  devtoolset-7-gcc-c++
+  devtoolset-7-toolchain
   libcurl-devel
   make
   patch
   python27-python-devel
   subversion-devel
   zlib-devel
+  wget
+  tar
 )
 build_centos6() {
   local mesos_version=$1 output_basedir=$2
   local python_outdir=$output_basedir/centos/6/python
   local image_name=centos:6
 
   pushd "$TMPDIR"
-    docker run -i -v $(pwd):/mesos-egg $image_name bash <<EOF
+    docker run -m=2g -i -v $(pwd):/mesos-egg $image_name bash <<EOF
```

Comment:
Any particular reason CentOS6 requires the docker container to be limited to 2gb's of memory? Would be worth putting this as a comment if that's the case 🙂

Reply:
Ok, Removed